crates/runtime/tests/docker/mod.rs

```diff
@@ -171,7 +175,7 @@ impl<'a> ContainerRunner<'a> {
                 format!("{container_port}/tcp"),
                 Some(vec![PortBinding {
                     host_ip: Some("127.0.0.1".to_string()),
-                    host_port: Some(format!("{host_port}/tcp")),
+                    host_port: Some(format!("{host_port}")),
```

Comment:
Oracle docker container was not exposing port when using `/tcp` suffix, tested all docker containers binds ports correctly with this change